app/views/facilities/_quicklinks.html.haml

```diff
@@ -0,0 +1,21 @@
+.quick_links 
+  .add_to_cart
+    - if not session_user
+      = link_to "#{t("admin.shared.add_to",model:t("pages.cart"))} (#{t("pages.login")})", new_user_session_path
+    - elsif session_user.can_override_restrictions?(product) || product.can_be_used_by?(acting_user)
+      - if not session_user.accounts.empty?
```

Comment:
In general, we prefer not to double negative (counting `empty?` as a negative). So this I would probably do as `if session_user.accounts.any?`. If you really want to use `empty?`, then I would do `unless session_user.accounts.empty?`